package.json

```diff
@@ -55,15 +55,18 @@
     "prismjs": "^1.17.1",
     "pug": "^2.0.4",
     "pug-cli": "^1.0.0-alpha6",
+    "pug-multiple-basedirs-plugin": "^0.1.5",
```

Comment:
it's just my personal opinion, but I'd prefer to have fixed version, because of transitive dependencies we can have with npm package

Reply:
I changed it for this dependency not others atm :)